woocommerce/woocommerce-gutenberg-products-block
# Add notice for outdated docs in Woo Blocks repo

## What

Fixes https://github.com/woocommerce/woocommerce/issues/42867

## Why

This repo is soon going to be archived, for anyone stumbling upon the documentation we want to alert them to this fact and let them know about the new document's location.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Check the changes I made, do they look OK?


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
